ql/src/test/results/clientpositive/llap/prepare_plan.q.out

```diff
@@ -128,11 +128,11 @@ STAGE PLANS:
       Processor Tree:
         ListSink
 
-PREHOOK: query: prepare pcount from select count(*) from src where key > ?
```

Comment:
The changes in the .q.out file are the expected ones however there seems to be something missing from the prepare statement implementation. After the changes introduced here the `query: prepare` pattern does not appear at all in the .q.out files. 

Is this normal? Should we log a follow-up ticket?

Reply:
Prepare queries doesn't go through the execution phase as per current implementation https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/Driver.java#L166C30-L166C44
So the pre-execution and post-execution hooks are skipped as well, which is why we miss seeing those results in the q.out file. I feel it is ok. Just that we need to use compilation side hooks to test the prepare queries. explain..prepare still shows up in the q.out files.
So I am not sure if we have to create a follow up to test the prepare queries? what is your take?

Reply:
Thanks for the explanation. Let's leave it as is at the moment and if really needed we can create a JIRA later ;)